siteimprove/includes/class-siteimprove.php

```diff
@@ -145,16 +145,17 @@ private function define_admin_hooks() {
 		$this->loader->add_action( 'siteimprove_before_settings_form', $plugin_admin, 'siteimprove_before_settings_form' );
 
 		// Siteimprove Actions.
-		if ( ! isset( $_GET['si_preview'] ) || '0' === $_GET['si_preview'] ) {
-			$this->loader->add_action( 'admin_init', $plugin_admin, 'siteimprove_init' );
-			$this->loader->add_action( 'publish_page', $plugin_admin, 'siteimprove_save_session_url_post' );
-			$this->loader->add_action( 'publish_post', $plugin_admin, 'siteimprove_save_session_url_post' );
-			$this->loader->add_action( 'edit_term', $plugin_admin, 'siteimprove_save_session_url_term', 10, 3 );
-			$this->loader->add_action( 'create_term', $plugin_admin, 'siteimprove_save_session_url_term', 10, 3 );
-			$this->loader->add_action( 'transition_post_status', $plugin_admin, 'siteimprove_save_session_url_product', 10, 3 );
-			$this->loader->add_action( 'wp_head', $plugin_admin, 'siteimprove_wp_head' );
-			$this->loader->add_action( 'admin_bar_menu', $plugin_admin, 'add_prepublish_toolbar_item', 500, 1 );
+		$this->loader->add_action( 'admin_init', $plugin_admin, 'siteimprove_init' );
```

Comment:
Because we don't need to have si_preview specific anymore. What we do instead now is inverting it, and if we are in si_preview, then we include the JavaScript file for emptying the wp-admin bar